Mathlib/Order/InitialSeg.lean

```diff
@@ -219,12 +285,12 @@ structure PrincipalSeg {α β : Type*} (r : α → α → Prop) (s : β → β 
   /-- The image of the order embedding is the set of elements `b` such that `s b top` -/
   down' : ∀ b, s b top ↔ ∃ a, toRelEmbedding a = b
 
--- Porting note: deleted `scoped[InitialSeg]`
-/-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≺i s` is an order
-embedding whose range is an open interval `(-∞, top)` for some element `top` of `β`. Such order
-embeddings are called principal segments -/
+@[inherit_doc]
 infixl:25 " ≺i " => PrincipalSeg
 
+/-- A `PrincipalSeg` between the `<` relations of two types. -/
+notation:25 α:24 " <i " β:25 => @PrincipalSeg α β (· < ·) (· < ·)
```

Comment:
Don't you want all the le_iff_le; lt_iff_lt lemmas here too?

Reply:
Nice catch! Yeah, I forgot about these. Fortunately it's just a bunch of one liners, so I'll bunch all of these theorems together at the end of the file.